### PR TITLE
feat: adding intensity of snow

### LINKED ISSUE #3: Add intensity option
The plugin copied the @catdad/canvas-confetti default snow example. 

It could be cool if one could choose between `light-snow`, `default` and `blizzard` for instance...

### PR DESCRIPTION
**Pull request by inezabonte**
closes #3

### CODE REVIEW

**Comment by raae** [comment]
Exited for this, will have a closed look later today 🎉

**Comment by inezabonte** [comment]
I would have attached gifs but my screen recording looks slower than it actually is

File: plugin/gatsby-node.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ exports.pluginOptionsSchema = ({ Joi }) => {
     colors: Joi.array()
       .items(Joi.string().default("#ffffff"))
       .description("Array of hex color values"),
+    intensity: Joi.string().valid('light-snow', 'blizzard'),
```
Comment (raae) [inline plugin/gatsby-node.js]: **nitpick**: Let's rename "light-snow" to just "light"
**nitpick**: Let's rename "default" to "regular", make it a valid option and make it the default.

**Review by raae — CHANGES_REQUESTED** [review]
The "blizzard" is so much fun ❄️

File: plugin/gatsby-node.js (hunk 1 above)
Comment (inezabonte) [reply plugin/gatsby-node.js]: Done ✅

> The "blizzard" is so much fun ❄️

Glad you like it 😄

File: README.md [hunk 2]
```diff
@@ -40,7 +40,8 @@ You can pass an array of colors to use via the plugin options
 ```
 
 You can choose the intensity of the snow.
-values; "blizzard", "light-snow"
+values; "regular", "blizzard", "light"
```
Comment (raae) [inline README.md]: **nitpick:** Could you change the text to

Accepted values in increasing intensity: "light", "regular" and "blizzard".  
Default value if non given: "regular"

**Review by raae — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (inezabonte) [reply README.md]: Sure thing ✅

**Comment by raae** [comment]
🎉

**Pull request merged**